lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/GeneralGroupCommunicationMessage.cs

```diff
@@ -15,58 +15,61 @@
 // specific language governing permissions and limitations
 // under the License.
 
+using System;
+
 namespace Org.Apache.REEF.Network.Group.Driver.Impl
 {
     /// <summary>
     /// Messages sent by MPI Operators. This is the class inherited by 
     /// GroupCommunicationMessage but seen by Network Service
     /// </summary>
-    public class GeneralGroupCommunicationMessage
-    {        
-        /// <summary>
-        /// Empty constructor to allow instantiation by reflection
-        /// </summary>
-        protected GeneralGroupCommunicationMessage()
-        {
-        }
-
+    public abstract class GeneralGroupCommunicationMessage
```

Comment:
Right, but a `GeneralGroupCommunicationMessage` should not be instantiated on its own, and hence should either be `abstract` or an `interface`. The getters are all inherited.